cmd/snap-confine/udev-support.c

```diff
@@ -197,6 +204,44 @@ void setup_devices_cgroup(const char *security_tag, struct snappy_udev *udev_s)
 	for (int i = 0; static_devices[i] != NULL; i++)
 		run_snappy_app_dev_add(udev_s, static_devices[i]);
 
+	// nvidia modules are proprietary and therefore aren't in sysfs and
+	// can't be udev tagged. For now, just add existing nvidia devices to
+	// the cgroup unconditionally (AppArmor will still mediate the access).
+	// We'll want to rethink this if snapd needs to mediate access to other
+	// proprietary devices.
+	struct stat sbuf;
+	char nv_path[15] = { 0 };	// /dev/nvidiaXXX
+	const unsigned nv_major = 195;
```

Comment:
Could you move the comment below (the one with URL to devices.txt) here and perhaps say something like this:
```
Major number of nvidia as set in stone by devices.txt from the kernel documentation.
URL
```

Reply:
Done